build.gradle.kts

```diff
@@ -71,8 +89,11 @@ tasks {
 
 publishing {
     publications {
-        create<MavenPublication>("default") {
+        create<MavenPublication>(publicationName) {
             from(components["java"])
+            groupId = group
+            artifactId = projectName
+            version = project.version.toString()
```

Comment:
Is it necessary to redefine default values ?

Reply:
mandatory from bintray plugin